### ISSUE TITLE
Unable to arm with override if 'use arm code' is disabled

### ISSUE THREAD

**Issue by ColColonCleaner** [nielsfaber/alarmo#803]
### Alarmo Version

1.9.10

### HA Version

2023.9.3

### Bug description

With 'use arm code' disabled, there doesn't appear to be a way to force arm the alarm if a sensor state is preventing the arm. Attempting to force arm via service call and passing a code that is configured as override is also blocked.

### Steps to reproduce

Have a sensor that will prevent the alarm from arming in its current state.
Disable 'use arm code'.
Attempt to override arm the alarm.

**Comment by ColColonCleaner** [comment]
Is there any additional info I should provide in order to have this looked into?
I like being able to arm the alarm without a code, but that option makes force-arming the alarm impossible as far as I can tell.

**Comment by nielsfaber** [comment]
I don't agree with these claims:
>  there doesn't appear to be a way to force arm the alarm if a sensor state is preventing the arm.

> Attempting to force arm via service call and passing a code that is configured as override is also blocked.

> I like being able to arm the alarm without a code, but that option makes force-arming the alarm impossible as far as I can tell.

Simple demonstration:

![image](https://github.com/nielsfaber/alarmo/assets/16861605/b4fdacf2-0110-4225-933b-b2576d50e656)

The red dots in the buttons indicate the alarm is not ready for arming (it would fail).
In this case, due to the backdoor being open (see toggle switches on the left).

Note that the green/red dots is a new feature to be added in the next version of alarmo-card.

Clicking the arm home button gives:
![image](https://github.com/nielsfaber/alarmo/assets/16861605/f9979c57-6e44-4e00-bd9f-099192a40e69)

Now, I select the 'bypass open sensors' option:
![image](https://github.com/nielsfaber/alarmo/assets/16861605/37b77e41-e7e6-4243-9a8f-7cb51eff8dc0)

I click again the 'arm home' button, result:
![image](https://github.com/nielsfaber/alarmo/assets/16861605/30a715c3-8626-4831-baa3-726e27db4e66)

To confirm the sensor is indeed bypassed:
![image](https://github.com/nielsfaber/alarmo/assets/16861605/025964c1-4d6b-41ea-aa1e-37d4642c0c3d)

All is as I would expect, no arming code needed.
Applying the 'bypass open sensors' setting via alarmo-card corresponds to setting `force: true` in the service `alarmo.arm`.
The same is also possible via the [actionable notification](https://github.com/nielsfaber/alarmo#actionable-notifications).

To summarise, there doesn't seem to be a bug here. Please provide some details/evidence to support your claims if you think there is.

**Comment by ColColonCleaner** [comment]
@nielsfaber I didn't know there was a special Alarmo card that allowed for the bypass. I was using the standard alarm card and mushroom alarm card and they don't have that functionality. This should resolve my problem. Thanks!

**Comment by ColColonCleaner** [comment]
As for the force arm via service call, i was attempting to do it by passing in a code that was configured as force arm. Didn't know that the 'force' parameter existed on the service, saw code as the only parameter.
